virtual_ecosystem/example_data/config/data_config.toml

```diff
@@ -1,5 +1,5 @@
 [core.data_output_options]
-save_initial_state = false
+save_initial_state = true
```

Comment:
Is this needed to get the display to work on the website?

Reply:
It's needed to generate those plots from the initial data! We could just point back to the original files, but it seems worth demonstrating that step.